src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+@import "./utils/functions";
+@import "./utils/variables.scss";
+
 body {
   margin: 0;
+  padding: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: getLimitSize($day-box-size , $boxGap);
+  padding: 10px;
+  margin: 0 auto;
+
+  @each $day , $margin in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $margin;
+    }
+  }
+
+  @for $lastDay from 28 to 31 {
+    &--month-length--#{$lastDay} :nth-child(n + #{$lastDay+1}) {
+      display: none;
+    }
+  }
+
+  &__box {
+    position: relative;
+    width: $day-box-size;
+    height: $day-box-size;
+    box-sizing: border-box;
```

Comment:
move `box-sizing: border-box;`

Reply:
move where , sorry but i didn't understand

Reply:
Each day is a grey (#eee) 100px square (including 1px black border)

According to checklist  we need border box here.

Reply:
`box-sizing: border-box;` should be inside `*` selector